examples/jenkins/pipeline/maven-pipeline.yaml

```diff
@@ -98,15 +98,30 @@ objects:
                 node {
                   stage("Build Image") {
                     unstash name:"war"
-                    sh "oc start-build ${appName}-docker --from-file=target/ROOT.war -n ${project}"
+                    def status = sh(returnStdout: true, script: "oc start-build ${appName}-docker --from-file=target/ROOT.war -n ${project}")
+
+                    def result = status.split("\n").find{ it.matches("^build.*started") }
+                    
+                    if(!result) {
+                      echo "Error !!!! No build started found"
```

Comment:
Can you simplify the error message: echo "ERROR: No started build found for ${appName}"